### ISSUE TITLE
[QUESTION] You can't perform this action at this time on pull requests  from user @krobelus 

### ISSUE THREAD

**Issue by nonumeros** [mawww/kakoune#4864]
### Question

I thought that either commenting on an issue arising from a bug  that has been introduced after an otherwise functioning kakoune revision is allowed on the kakoune project. 

Also, I thought, naively thought that any Pull  Request from a contributor to kakoune would improve the editor, and not degrade it.

I just found out that I can't comment or give a like :) on  @krobelus 

![Screenshot_2023-03-07_02-49-22](https://user-images.githubusercontent.com/11535575/223360648-97693d3b-ab13-4d03-b669-4ed3b1a86cd9.png)

or

![Screenshot_2023-03-07_03-04-26](https://user-images.githubusercontent.com/11535575/223361577-23250dee-7c1c-4a5e-b0db-fdd2abab1472.png)

or 

![Screenshot_2023-03-07_03-05-56](https://user-images.githubusercontent.com/11535575/223361617-c1930bb1-df1a-484d-99cf-95f62b6a340e.png)

Googling around this I came across that the message from github about `You can't perform this action at this time.` 

It states that the repository owner has blocked the specific user and hence the message :+1: 

 :1st_place_medal: 

https://stackoverflow.com/a/67354715

![Screenshot_2023-03-07_03-08-06](https://user-images.githubusercontent.com/11535575/223362347-7fa5cabd-bba8-48d0-8b78-e16d4d17c29b.png)

![Screenshot_2023-03-07_02-52-06](https://user-images.githubusercontent.com/11535575/223364636-440eddb1-9588-4892-84f6-fe499476473e.png)

**Comment by nonumeros** [comment]
:) it is what it is I guess \o/

**Comment by nonumeros** [comment]
@mawww  unless the code implementation or the pull request is intentional buggy, I guess one must use the revision that you committed after merging changes into the master branch

**Comment by nonumeros** [comment]
> @mawww  unless the code implementation or the pull request is intentionally buggy, I guess one must use the revision that you committed after merging changes into the master branch. I or any other user in that case would not be able to even comment on the other user's pull request of a new branch


@mawww  As evidenced by the following @krobelus introduced a pull request that created  havoc , see: 

https://github.com/mawww/kakoune/issues/4775

You closed the issue and merged the new pull request with 

https://github.com/mawww/kakoune/commit/59b8b99577ce38bc2b3cad330c1108eeb4faa447

**Comment by nonumeros** [comment]
@mawww  the only explanation I can think of by @krobelus blocking me off on any pull request, or on any other issue or whatever that @krobelus opened, and on a very underhanded and  sneaky fashion , it follows that it is entirely intentional.  On purpose.

**Comment by nonumeros** [comment]
About blocking users
You can block a user in your account settings or from the user's profile. GitHub will not notify the user when you block them. If you want to avoid contributing to the same project as someone you've blocked, you can choose to display a warning on any repositories with prior contributions from a blocked user. For more information, see "[Blocking a user in your account settings](https://docs.github.com/en/communities/maintaining-your-safety-on-github/blocking-a-user-from-your-personal-account#blocking-a-user-in-your-account-settings)." You may still see the activity of blocked users in shared spaces and blocked users can delete their existing content.

Tip: If you're blocking a user because of a heated conversation, consider locking the conversation so only collaborators can comment. For more information, see "[Locking conversations](https://docs.github.com/en/communities/moderating-comments-and-conversations/locking-conversations)."

When you block a user:

The user stops following you
The user stops watching and unpins your repositories
The user's stars and issue assignments are removed from your repositories
The user's votes on discussions or comments in your repositories are deleted
The user is removed as a collaborator on your repositories
The user's contributions to your repositories are no longer counted as contributions for them
Your contributions to the blocked user's repositories are no longer counted as contributions for you
You are removed as a collaborator on their repositories
Their sponsorship of you is canceled
Any pending repository or account successor invitations to or from the blocked user are canceled
The user is removed as a collaborator from all the projects and classic projects owned by you
You are removed as a collaborator from all the projects and classic projects owned by the user

After you've blocked a user, they cannot:

Comment on or edit issues or pull requests that you've created
React to your comments on issues, pull requests, and commits
Follow you or see your content in their activity feed
Assign you to issues or pull requests
Invite you as a collaborator on their repositories
Invite you as a collaborator on a security advisory
Cross-reference your repositories in comments
Fork, watch, pin, or star your repositories
Sponsor you
Add you as a collaborator on their projects and classic projects
Make changes to your public projects and classic projects
In repositories you own, blocked users also cannot:

Open issues
Send, close, or merge pull requests
Comment on issues, pull requests, or commits

https://docs.github.com/en/communities/maintaining-your-safety-on-github/blocking-a-user-from-your-personal-account#about-blocking-users

**Comment by nonumeros** [comment]
@mawww  The last interaction I can remember having on a Pull Request from @krobelus and one which you participated was in https://github.com/mawww/kakoune/pull/4824 

It was about the bracketed paste mode which even ncurses devs had worked on it recently 

But I can't pinpoint that @krobelus was instructed by you, to block me off either. Even if it were true, and you asked @krobelus to block me off from any participation, (find it unlikely) but worth mentioning as a remote possibility, in practice: the interface of github allows any user to do as they wish.; and block off other users, simply because it can. So be it. :) I guess.

**Comment by krobelus** [comment]
Your contributions are appreciated but you talk a lot while saying very little. Please try to empathize with your readers who have limited time and context. My intention was to unsubscribe from notifications chiefly. I'll unblock you, let's see if that works out

**Comment by nonumeros** [comment]
@krobelus  there you are! 

I'm not going to go back and forth with you here. 

You were petty because I brought up the issue https://github.com/mawww/kakoune/issues/4775 and you felt that the code you had put together was fault free.
